cfg/Astra.cfg

```diff
@@ -35,6 +35,7 @@ gen.add("depth_registration", bool_t, 0, "Depth data registration", True)
 gen.add("color_depth_synchronization", bool_t, 0, "Synchronization of color and depth camera", False)
 gen.add("auto_exposure", bool_t, 0, "Auto-Exposure", True)
 gen.add("auto_white_balance", bool_t, 0, "Auto-White-Balance", True)
+gen.add("exposure", int_t, 0, "Exposure time in ms", 0, 0, 255)
```

Comment:
Are you sure the time units are `ms`?  Some cameras use the full byte range to represent a smaller/larger range (in engineering units).

Reply:
Good catch.  I could not find any documentation to support what the units are.  Is it sufficient to just remove the units since we do not know?